runconfig/parse.go

```diff
@@ -98,6 +98,7 @@ func Parse(cmd *flag.FlagSet, args []string) (*Config, *HostConfig, *flag.FlagSe
 		flSwappiness        = cmd.Int64([]string{"-memory-swappiness"}, -1, "Tuning container memory swappiness (0 to 100)")
 		flNetMode           = cmd.String([]string{"-net"}, "default", "Set the Network for the container")
 		flMacAddress        = cmd.String([]string{"-mac-address"}, "", "Container MAC address (e.g. 92:d0:c6:0a:29:33)")
+		flIp4Addresses      = cmd.String([]string{"-ip-address"}, "", "Container IP4 addresses separated by comma (e.g. 10.4.4.2, 10.2.2.2)")
```

Comment:
Before the --ip-address param would receive something like 10.4.4.2/32. What is it expecting now for a single address? The same? And for multiple addresses? is it addr1/32,addr2/32 ?or addr1,addr2/32 ?